migrations/2023-01-19-122511_add_refund_error_code/up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE refund
+ADD IF NOT EXISTS refund_error_code VARCHAR(255) DEFAULT NULL;
```

Comment:
Do you require this to be `VARCHAR(255)`, or can it be of a smaller size?

Reply:
Not sure since connector like checkout sends error message like in error code format

Reply:
TEXT